rickchurch/templates/base.html

```diff
@@ -2,16 +2,15 @@
 <html lang="en">
 
 <head>
-  {% block head %}
-  <title>rickchurch</title>
+  <title>{% block title %}{% endblock %}</title>
```

Comment:
Replacing this with `<title>Church of Rick • {% block title %}{% endblock %}</title>` would get rid of the duplication in the different templates
edit: We could also call it subtitle if that makes more sense

Reply:
Yeah, I wanted to do that but since it was named `title` I didn't. with `subtitle` it makes much more sense, I agree, I'll changed this.